driver/sql/postgres/eventstore_test.go

```diff
@@ -297,7 +301,15 @@ func TestEventStore_Load(t *testing.T) {
 				factory.EXPECT().CreateEventStream(gomock.AssignableToTypeOf(&sql.Rows{})).Return(expectedStream, nil).Times(1)
 
 				strategy := mockSQL.NewPersistenceStrategy(ctrl)
-				strategy.EXPECT().ColumnNames().Return(columns).AnyTimes()
+
+				if matcher != nil {
+					strategy.EXPECT().PrepareSearch(matcher).Return([]byte(" AND version > $2 AND version < $3"), []interface{}{}).Times(1)
+				} else {
+					strategy.EXPECT().PrepareSearch(matcher).Return([]byte{}, []interface{}{}).AnyTimes()
+				}
+
+				strategy.EXPECT().InsertColumnNames().Return(columns).AnyTimes()
+				strategy.EXPECT().EventColumnNames().Return(columns).AnyTimes()
```

Comment:
You may want to change the Event and/or insert columns since they both are the same at the moment a test may pass that by accident calls `InsertColumnNames` when it should call `EventColumnNames`